test/Microsoft.Health.Dicom.Web.Tests.E2E/Common/InProcTestDicomWebServer.cs

```diff
@@ -138,5 +149,23 @@ private static string GetProjectPath(string projectRelativePath, Type startupTyp
 
             throw new Exception($"Project root could not be located for startup type {startupType.FullName}");
         }
+
+        private static IHost CreateWebJobsHost<T>(string path)
+            where T : FunctionsStartup, new()
+        {
+            string contentRoot = GetProjectPath(path, typeof(T));
+            return new HostBuilder()
+                .UseContentRoot(contentRoot)
+                .ConfigureLogging(b => b.AddConsole())
+                .ConfigureAppConfiguration(b => b
+                    .Add(AzureFunctionsConfiguration.CreateRoot())
+                    .Add(new HostJsonFileConfigurationSource(contentRoot))
+                    .Add(EnvironmentConfig.FromLocalSettings(contentRoot)))
+                .ConfigureWebJobs((c, b) => b
+                    .UseWebJobsStartup(typeof(T), new WebJobsBuilderContext { Configuration = c.Configuration }, NullLoggerFactory.Instance)
+                    .AddDurableTask())
+                .ConfigureServices(s => s.Configure<HostOptions>(h => h.ShutdownTimeout = Timeout.InfiniteTimeSpan))
```

Comment:
Won't it be better to set a timeout here?

Reply:
Yeah, and by default I think the timeout is 5 seconds, so that may very well be sufficient